source/server/nierr_Status.cpp

```diff
@@ -0,0 +1,106 @@
+/*!
```

Comment:
What is new code here vs what is imported? And why? Is there a way to avoid mixing new code and imported code?

Reply:
As discussed with @achanyou this code is all "imported" but from multiple source files (i.e., with dependencies inlined) and not always in their entirety. This was because trying to pull these in as a whole would pull in a long chain of deps. Pulling in all the deps may be fine in many situations, but it has a different set of challenges problems and I'm happy with how Ariane has handled this.

We should add comments to the top of these files explaining where they came from. And put in-line comments on any sections that are inlined from dependencies. I also think that they should move back to the `imports` directory instead of `source/server`. Neither location is perfect, but that's a better representation of where it came from.